src/middlewares/validationHandler/guestValidator.js

```diff
@@ -0,0 +1,25 @@
+import { check, validationResult } from "express-validator";
+
+exports.guestValidator = [
```

Comment:
The variable should be in snake case, this is an array not a function. Do you get?

Reply:
Yes I have modified it.